<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordsWindow</class>
 <widget class="QWidget" name="RecordsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>借阅记录</string>
  </property>
  <property name="whatsThis">
   <string>查看图书管理系统的借阅记录。</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#centralWidget
{
	border: 1px solid rgb(235,235,235);
	background-color: white;
}</string>
     </property>
     <layout class="QVBoxLayout" name="centralLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="RecordsTitleWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#RecordsTitleWidget
{
	background-color: white;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="RecordsTitle">
           <property name="styleSheet">
            <string notr="true">#RecordsTitle
{
	font: bold 14px &quot;宋体&quot;;
}</string>
           </property>
           <property name="text">
            <string>借阅记录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SwitchMode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>切换到管理员模式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RefreshRecords">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>15</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#RefreshRecords
{
	border: 0;
	image: url(:/Icon/refresh_white.png);
}
#RefreshRecords:pressed
{
	image: url(:/Icon/refresh_black.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="SearchLineWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit, QPushButton
{
	font-size: 16px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="SearchWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#SearchWidget
{
	border-bottom: 2px solid rgb(243, 243, 243);
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="SearchType">
              <property name="styleSheet">
               <string notr="true">#SearchType
{
	border: 0;
	color: gray;
}
#SearchType:pressed
{
	color: black;
}</string>
              </property>
              <property name="text">
               <string>搜索类型</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Keywords">
              <property name="styleSheet">
               <string notr="true">#Keywords
{
	border: 0;
}</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SearchButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#SearchButton
{
	border:0;
	image:url(:/Icon/zoom.png);
	padding:5px;
	background-color:rgba(255,255,255,0);
}
#SearchButton:hover
{
	background-color:rgba(0,0,0,30);
}
#SearchButton:pressed
{
	background-color:rgba(0,0,0,60);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="RecordsList">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SearchButton</sender>
   <signal>clicked()</signal>
   <receiver>RecordsWindow</receiver>
   <slot>SearchEvent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RefreshRecords</sender>
   <signal>clicked()</signal>
   <receiver>RecordsWindow</receiver>
   <slot>listAllRecords()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Keywords</sender>
   <signal>returnPressed()</signal>
   <receiver>RecordsWindow</receiver>
   <slot>SearchEvent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SwitchMode</sender>
   <signal>clicked()</signal>
   <receiver>RecordsWindow</receiver>
   <slot>onSwitchModeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SearchEvent()</slot>
  <slot>listAllRecords()</slot>
  <slot>onSwitchModeClicked()</slot>
 </slots>
</ui>
